### PR TITLE
Use partial class

### PR DESCRIPTION
**Pull request by jsturtevant**
makes generation simpler by not needing reflection

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -401,4 +401,6 @@ FodyWeavers.xsd
 **/**target
 
 # VSCode
-**/**.vscode/
\ No newline at end of file
+**/**.vscode/
+
+MyProject/adder.wit
```
Comment (marosset) [inline .gitignore:406]:
```suggestion
MyProject/adder.wit

```

**Review by marosset — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (danbugs) [inline .gitignore:406]:
```suggestion
```

Did you move locations of this to `MyProject`? In main, I have it in root and the README uses that to run the demo.

**Review by danbugs — APPROVED** [review]
Woah – This is super simple and clean, good stuff!

File: .gitignore (hunk 1 above)
Comment (jsturtevant) [reply .gitignore:406]: no, this just avoids adding it here if you follow the demo and copy it to this location

Comment (danbugs) [reply .gitignore:406]: The demo doesn't copy it, though – it uses it from root!

Comment (jsturtevant) [reply .gitignore:406]: ah, yea your right, I think I copied it there when I was messing around.  I can drop this

**Pull request merged**